### DISCUSSION TITLE
Welcome to Himmelblau Discussions!

### DISCUSSION THREAD

**Discussion by dmulder** [himmelblau-idm/himmelblau#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vaughanjackson88** [comment]
Hi, Firstly Himmelblau works great!! 
Was wondering if I could change the url on initial login/MFA setup to a QR code?

This would just make it that little bit easier to onboard new users and get their MS Authenticator setup.

Cheers

**Reply by dmulder** [reply]
Yes, this is possible, but I haven't implemented it yet. Would you file an enhancement request? I can probably get that included in version 1.x.

**Reply by vaughanjackson88** [reply]
Sure can.. Any idea when that kind of feature would be released? Just as we have a 1000+ rollout happening soon :)

Thanks Again
